tests/test_axis.py

```diff
@@ -140,60 +149,65 @@ class TestCirculationMonitor(DatabaseTest):
     )
 
     def test_process_book(self):
-        monitor = Axis360CirculationMonitor(self._db)
-        monitor.api = None
-        edition, license_pool = monitor.process_book(
-            self.BIBLIOGRAPHIC_DATA, self.AVAILABILITY_DATA)
-        eq_(u'Faith of My Fathers : A Family Memoir', edition.title)
-        eq_(u'eng', edition.language)
-        eq_(u'Random House Inc', edition.publisher)
-        eq_(u'Random House Inc2', edition.imprint)
-
-        eq_(Identifier.AXIS_360_ID, edition.primary_identifier.type)
-        eq_(u'0003642860', edition.primary_identifier.identifier)
-
-        [isbn] = [x for x in edition.equivalent_identifiers()
-                  if x is not edition.primary_identifier]
-        eq_(Identifier.ISBN, isbn.type)
-        eq_(u'9780375504587', isbn.identifier)
-
-        eq_(["McCain, John", "Salter, Mark"], 
-            sorted([x.name for x in edition.contributors]),
-        )
-
-        subs = sorted(
-            (x.subject.type, x.subject.identifier)
-            for x in edition.primary_identifier.classifications
-        )
-        eq_([(Subject.BISAC, u'BIOGRAPHY & AUTOBIOGRAPHY / Political'), 
-             (Subject.FREEFORM_AUDIENCE, u'Adult')], subs)
-
-        eq_(9, license_pool.licenses_owned)
-        eq_(8, license_pool.licenses_available)
-        eq_(0, license_pool.patrons_in_hold_queue)
-        eq_(datetime.datetime(2015, 5, 20, 2, 9, 8), license_pool.last_checked)
-
-        # Three circulation events were created, backdated to the
-        # last_checked date of the license pool.
-        events = license_pool.circulation_events
-        eq_([u'title_add', u'check_in', u'license_add'], 
-            [x.type for x in events])
-        for e in events:
-            eq_(e.start, license_pool.last_checked)
-
-        # A presentation-ready work has been created for the LicensePool.
-        work = license_pool.work
-        eq_(True, work.presentation_ready)
-        eq_("Faith of My Fathers : A Family Memoir", work.title)
-
-        # A CoverageRecord has been provided for this book in the Axis
-        # 360 bibliographic coverage provider, so that in the future
-        # it doesn't have to make a separate API request to ask about
-        # this book.
-        records = [x for x in license_pool.identifier.coverage_records
-                   if x.data_source.name == DataSource.AXIS_360
-                   and x.operation is None]
-        eq_(1, len(records))
+        with temp_config() as config:
```

Comment:
This is fine for now but you don't literally need to set up a LocalAnalyticsProvider here, just to maintain the previous functionality. It'd be a bit nicer if you used a MockAnalyticsProvider instead, and verified that the appropriate events were recorded in the MockAnalyticsProvider. (This would require changing MockAnalyticsProvider so that it kept a log of all the events it saw, not just the most recent one.)